### PR TITLE
use splatting in BitArray

### PR DESCRIPTION
**Pull request by frankschae**
According to @mzgubic suggestion from https://github.com/JuliaDiff/ChainRules.jl/pull/506 .

### CODE REVIEW

**Comment by frankschae** [comment]
Oh the version test is failing. Should this get a "1.10.1"?

**Comment by mzgubic** [comment]
Thanks! Yes this is a "bug fix" so patch should be bumped (major.minor.patch) according to SemVer. The previous PR was an addition of a rule so minor was bumped :)

File: Project.toml
Comment (mzgubic) [inline Project.toml:4]:
```suggestion
```

**Review by mzgubic — APPROVED** [review]

**Comment by frankschae** [comment]
(Sorry for the two steps. I forgot to pull before making this PR.)

**Comment by oxinabox** [comment]
@mzgubic  as  frankschae does not have merge rights on the repo,
you should merge the PR when you accept it, (assuming nothing minor is pending)

**Comment by mzgubic** [comment]
oh yeah my bad, was waiting for CI to pass but then forgot

**Pull request merged**
